libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxVoiceInstructionsPlayer.kt

```diff
@@ -92,9 +92,11 @@ class MapboxVoiceInstructionsPlayer @JvmOverloads constructor(
                 abandonFocus()
             }
 
-            val currentAnnouncement = currentPlayCallback.announcement
-            val currentClientCallback = currentPlayCallback.consumer
-            currentClientCallback.accept(currentAnnouncement)
+            if (currentPlayCallback != null) {
```

Comment:
We should also add a changelog entry. Sth like:
```
- Fixed a rare `java.lang.NullPointerException: Attempt to read from field 'en.b dn.m.a' on a null object reference` crash in `PlayCallback.getAnnouncement`. [#6760](https://github.com/mapbox/mapbox-navigation-android/pull/6760)
```
to `mapbox-navigation-android/CHANGELOG.md`.